documentation/src/docs/getting-started/quick-start.md

```diff
@@ -0,0 +1,59 @@
+# Quick Start Guide
+
+## Download the latest Pravega
+```
+wget https://github.com/pravega/pravega/releases/download/v0.9.0-rc0/pravega-0.9.0.tgz
+tar zxvf pravega-0.9.0.tgz
+cd pravega
+```
+
+## Start Pravega standalone cluster
+```
+./bin/pravega-standalone
+```
+
+## Create your scope and your stream
```

Comment:
Same comment here about Pravega abstractions (Stream, Scope,...). In most of the documentation we write them starting with caps.